apps/mobile/src/components/TradeChecklistFlow/atoms/index.ts

```diff
@@ -0,0 +1,99 @@
+import {atom, type SetStateAction, type WritableAtom} from 'jotai'
+import {UnixMilliseconds} from '@vexl-next/domain/dist/utility/UnixMilliseconds.brand'
+import getValueFromSetStateActionOfAtom from '../../../utils/atomUtils/getValueFromSetStateActionOfAtom'
+import {DateTime} from 'luxon'
+import {type AvailableDateTimeOption} from '../../../state/tradeChecklist/domain'
+
+export const availableDateTimesAtom = atom<AvailableDateTimeOption[]>([])
+
+export function createTimeOptionAtomForTimeToDropdown(
+  timestampTo: UnixMilliseconds
+): WritableAtom<UnixMilliseconds, [SetStateAction<UnixMilliseconds>], void> {
+  return atom(
+    (get) =>
+      get(availableDateTimesAtom).find(
+        (dateTime) => dateTime.to === timestampTo
+      )?.to ?? UnixMilliseconds.parse(DateTime.now().toMillis()),
```

Comment:
```suggestion
      )?.to ?? unixMillisecondsNow(),
```

Better to use the specified function for this